projections/azure-samples/CONTRIBUTING.md

```diff
@@ -51,12 +51,12 @@ chances of your issue being dealt with quickly:
 * **Suggest a Fix** - if you can't fix the bug yourself, perhaps you can point to what might be
   causing the problem (line of code or commit)
 
-You can file new issues by providing the above information at the corresponding repository's issues link: https://github.com/[organization-name]/[repository-name]/issues/new].
+You can file new issues by providing the above information at the corresponding repository's issues link: <https://github.com/[organization-name]/[repository-name]/issues/new>.
```

Comment:
this shouldn't require the `<>` since it is an example of how to find the new issue section for the users desired repo. if anything this should have tick marks around it to get it to pop in the markdown.

Reply:
Sure thing, it would be a link though as soon as someone puts in a value.  So was getting it ready for them so they didn't need to first remove the \` and then figure out the correct syntax to use for a url.

Any thoughts on the usibility aspect?  Which is better?  The end result should be a link that they can click, so do we want users to figure out the syntax?  I can turn it into []() and it will have the same effect as the \<>, but would still violate your thoughts here.

Reply:
Addressed in [`fcf0603` (#17)](https://github.com/microsoft/repo-templates/pull/17/commits/fcf0603b43030678c014bb6fb8fe6f143030c63a)